src/Server/setup_sharify.ts

```diff
@@ -80,10 +80,13 @@ sharify.data = _.extend(
     "SALESFORCE_CHAT_INSTANCE_URL",
     "SALESFORCE_CHAT_HELP_URL",
     "SALESFORCE_CHAT_ORG_ID",
-    "SALESFORCE_CHAT_EMBEDDED_SERVICE_NAME",
+    "SALESFORCE_CHAT_EMBEDDED_SERVICE_COLLECTOR_NAME",
```

Comment:
I have no idea what name fits here, it's more a question to clarify, is `COLLECTOR` a good name to differentiate from `AUCTION`?

Reply:
Good question. This is because now we'll have two different Salesforce services being used on artsy.net.
So far we were using the "Collector Support Chat" for everything.

With these changes, we have "Collector Support Chat" and "Auction Support Chat" being used depending on the scenario.

Reply:
thanks for explaining, so maybe we can make it clearer since in both cases they work for the collector but 1 for auction scope and the other for the general scope.
maybe `GENERAL`/`COMMON` instead of `COLLECTOR`?

Reply:
I just wonder how confusing could it be for future maintainers to use a different terminology than just matching the same we're already using on Salesforce, which is the actual source of truth for this stuff...

Reply:
Was more curious about other possibilities, but let's move with this and re-visit later if we need it.

Reply:
Cool, thank you!! 💜